overlord/snapstate/snapstate.go

```diff
@@ -227,6 +230,124 @@ func Install(s *state.State, name, channel string, userID int, flags Flags) (*st
 	return doInstall(s, &snapst, ss)
 }
 
+// like strings.Contains but assumes the list is sorted
```

Comment:
This is not at all like strings.Contains.

Also, let's please respect the usual documentation conventions.